utils/readboundaries_shp.py

```diff
@@ -140,9 +210,12 @@ def get_wdb_boundaries(resolution,level,rivers=False):
     f2.close()
 
 for resolution in ['c','l','i','h','f']:
-    f = open('rivers_'+resolution+'.dat','wb')
-    f2 = open('riversmeta_'+resolution+'.dat','w')
-    for level in range(1,12):
+    f = open(os.path.join(OUTPUT_DIR, 'rivers_'+resolution+'.dat'), 'wb')
+    f2 = open(os.path.join(OUTPUT_DIR, 'riversmeta_'+resolution+'.dat'), 'w')
+    # Levels above 5 are intermittent rivers and irrigation canals.
+    # They haven't been included in the past, as far as I can tell, so I'm
+    # not including them here...
+    for level in range(1, 6):
         poly, polymeta = get_wdb_boundaries(resolution,level,rivers=True)
         offset = 0
```

Comment:
Sneaky little bugger... Apparently this script has never been used for processing and packaging basemap's data. The offset keeps getting reset for each level in this script, but it doesn't do that for the other readboundary.py script because it doesn't have different river levels in the dump.

If one just moves this offset to above the loop, the data produced by readboundary_shp.py works perfectly for `examples/hires.py` at all resolutions!

I'll create a new branch that has all the code work from @joferkington in a commit and credited to him as well, and a redo of the data packaging in a separate commit.